decidim-dev/lib/decidim/dev/test/rspec_support/upload.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+RSpec.configure do |config|
+  config.include ActionDispatch::TestProcess::FixtureFile
```

Comment:
I think we don't need to include this after #8681, right?

Can you merge with `develop` so that we'll see what changes are still needed after #8681?

I have few other comments in the works but I think this is pretty important to make sure at least that the Rails 6.1 update won't break the dynamic attachments.